setup.cfg

```diff
@@ -25,10 +25,10 @@ package_dir =
     =src
 include_package_data = True
 packages = find:
-python_requires = >= 3.10
+python_requires = >= 3.9.5
 install_requires =
     numpy==1.26.1
-    pandas==2.1.2
+    pandas[performance]==2.1.2
```

Comment:
what's pandas performance?

Reply:
Pandas performance has dependencies like `bottleneck `and `numba` which are needed to run RiskNet on DSMLP. Also, according to the pandas docs, these dependencies "provide speed improvements, especially when working with large data sets".